### ISSUE TITLE
Frontpage design

### ISSUE THREAD

**Issue by matiassingers** [javascriptmy/website#1]
IANAD (_I am not a designer_)

Let's try with some open-source style designing of the initial website.

---
### Inspiration
- [CopenhagenJS](http://copenhagenjs.dk/)
- [js.la](http://js.la/)
- [sandiego.js](http://sandiegojs.org/)
- [Manila.js](http://manilajs.com/)
- [Berlin.JS](http://berlinjs.org/)
- [FranceJS](http://francejs.org/)
- [LyonJS](http://lyonjs.org/)
- [NantesJS](http://nantesjs.org/)
- [PragueJS](http://www.praguejs.cz/)
- [JavaScript Zagreb](http://jszgb.com/)
- [Kopi.JS](http://kopijs.org/)

I quite like the simplistic approach that [CopenhagenJS](http://copenhagenjs.dk/), [Kopi.JS](http://kopijs.org/), [js.la](http://js.la/) and [Berlin.JS](http://berlinjs.org/) takes, basically just the following views:
- Frontpage
  - Logo, description
  - Next meetup details(time, location, attendees and agenda)
  - Organizers(contact info) and sponsors
  - Meta data(twitter, GitHub, etc.)
- List of past events
- Single meetup/event view
  - Details about the particular event(time, location, etc.)

---

I'm okay with just doing a quick prototype for the frontpage, but it would be great if anybody has some kind of creative skills (I definitely don't :stuck_out_tongue_winking_eye:)

@javascriptmy/core am I missing any important info?

**Comment by matiassingers** [comment]
I just quickly did a small mashup of [CopenhagenJS](http://copenhagenjs.dk/) and [Kopi.JS](http://kopijs.org/):

![frontpage](https://cloud.githubusercontent.com/assets/938453/6967305/029a4022-d991-11e4-81cf-9c0eb9bffba6.png)

It's just a quick [Balsamiq file](http://cl.ly/193s2O361d1B), if anybody wants to play around with it.

**Comment by soggie** [comment]
Comic. Sans. Do you have a death wish or something?

**Comment by matiassingers** [comment]
@soggie haha, it's actually not Comic Sans - and just the basic Balsamiq sketch mockup format.

Quite like the low fidelity of Balsamiq mockups, very easy to quickly piece things together.

---

Are we missing anything from that mockup?
Like super important features?

**Comment by soggie** [comment]
Looks fine so far, but I'd have the map in full width, front and center. I actually quite like BerlinJS' design, with empty slots as a call for action for speakers. I can do that actually. Maybe we need to integrate Ghost as well.

**Comment by yujinlim** [comment]
I think it would be cool to have speaker profile as well that would link to their personal github/fb/twitter pages.

**Comment by matiassingers** [comment]
Yeah I like the slots for speakers and profiles with social media links is a great idea!

For older events, it would also be cool to link to speakerdeck/slideshare or whatever if the slides are public.

**Comment by yujinlim** [comment]
+1 on speakerdeck/slides/video

**Comment by hueitan** [comment]
Perhaps adding a slack room to allow attendees for having some discussions?

**Comment by zulfajuniadi** [comment]
@huei90 we're already on slack. Ask access from @soggie.

**Comment by matiassingers** [comment]
@huei90 or check out: https://javascriptmy-slack.herokuapp.com/ - _(completely untested)_
